Project Management Process.md

```diff
@@ -0,0 +1,293 @@
+# IPFS Project Management Process
+
+**Status: work in progress, not even a full draft yet**
+
+## Introduction
+TODO: describe the purpose of this document
+
+- this is an opt-in process, nobody in the community is *required* to use this process
+
+## Table on Contents
+- [TL;DR](#tldr)
+- [Projects](#projects)
+- [Roles](#roles)
+  - [Product Owners](#product-owners)
+  - [Project Leads](#project-leads)
+  - [Project Managers](#project-managers)
+- [Structure](#structure)
+  - [Overview](#overview)
+  - [Goals](#goals)
+  - [Backlog](#backlog)
+  - [Milestones](#milestones)
+  - [Project Roadmap](#project-roadmap)
+  - [Organization Roadmap](#organization-roadmap)
+- [Flow](#flow)
+  - [Quarterly Planning](#quarterly-planning)
+  - [Software Development Pipeline](#software-development-pipeline)
+    - [Kanban Pipeline](#kanban-pipeline)
+- [Implementation](#implementation)
+
+## TL;DR
+
+The IPFS project management process has the following **cadence**:
+
+Roadmap Updates
+- every 3 months
+- identify, discuss and plan the next steps for the organization as a whole
+
+"Sprint" Updates
+- once a week (2 weeks?)
+- discuss last week's progress
+- discuss plan for the next week
+
+The IPFS project management process breaks down to the following **units of work**:
+
+Projects 
+- may contain other projects
+- have a project leader who "roadmaps" (decides what are goals and milestones, and when they should happen)
+- have a roadmap (list of milestones and goals)
+- have a backlog (list of goals)
+
+Milestones 
+- measurable unit of progress
+- contains a list of goals, ordered by priority
```

Comment:
@RichardLitt "goals" would be each 1step you take in that terminology. And goals would be huge and few (maybe 1-1 with projects?).

- I think it's ok to have milestones contain goals.
- i think "goal" is better than "task" because it makes it much more action oriented. there's a "desired end state". instead of "things to do".